examples/static_react_task/webapp/src/app.jsx

```diff
@@ -11,19 +11,76 @@ import ReactDOM from "react-dom";
 import { BaseFrontend, LoadingScreen } from "./components/core_components.jsx";
 import { useMephistoTask } from "mephisto-task";
 
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: null, errorInfo: null };
+  }
```

Comment:
I believe you can just put `state = { error: null, errorInfo: null }` outside of the constructor. this should let you avoid having to implement the constructor altogether and make this code a little more condensed. see more [here](https://www.robinwieruch.de/react-state-without-constructor) and [here](https://github.com/the-road-to-learn-react/react-alternative-class-component-syntax)